packages/bpk-component-image/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "name": "bpk-component-image",
+  "version": "0.0.0",
```

Comment:
Just want to confirm that `private` and `0.0.0` are the convention we use for unreleased components.